nimble/host/services/audio/pacs/include/services/pacs/ble_audio_svc_pacs.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef H_BLE_AUDIO_SVC_PACS_
+#define H_BLE_AUDIO_SVC_PACS_
+
+#define BLE_SVC_AUDIO_PACS_UUID16                                   0x1850
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_SINK_PAC                      0x2BC9
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_SINK_AUDIO_LOCATIONS          0x2BCA
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_SOURCE_PAC                    0x2BCB
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_SOURCE_AUDIO_LOCATIONS        0x2BCC
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_AVAILABLE_AUDIO_CONTEXTS      0x2BCD
+#define BLE_SVC_AUDIO_PACS_CHR_UUID16_SUPPORTED_AUDIO_CONTEXTS      0x2BCE
+
+int ble_svc_audio_pacs_set_sink_audio_loc(uint32_t audio_locations);
+int ble_svc_audio_pacs_set_source_audio_loc(uint32_t audio_locations);
+int ble_svc_audio_pacs_set_sup_contexts(uint16_t sink_contexts,
+                                        uint16_t source_contexts);
+int ble_svc_audio_pacs_set_avail_contexts(uint16_t conn_handle,
+                                          uint16_t sink_contexts,
+                                          uint16_t source_contexts);
```

Comment:
I wonder whether is there a need to have those 4 functions instead of 1 e.g. ble_svc_audio_pacs_init. As those are basically used to initialize the characteristic values. What's your opinion on that?

```
struct ble_svc_audio_pacs_init_param {
    uint32_t audio_locations;
    uint16_t available_contexts;
    uint16_t supported_contexts;
};

struct ble_svc_audio_pacs_init(uint8_t flags, struct ble_svc_audio_pacs_init_param *param);
```

where `flags ` are the ones from Codec API`. Hmm, this makes me feel that those should not be flags, but rather `type` with 2 possible values Sink and Source defined as enum probably so that we could reuse it in different services.

Reply:
One problem I see is that `available_contexts` are per connection - so they cannot be set on service init, but after connection is established. So we have one exception. And because in previous comment you asked to disallow re-setting these values, because there is no notifications support for them (which I 100% agree with), we would have to split these into these that init on startup and `available_contexts`. On the other hand we here, as you notice, SRC/SNK pairs, and there will be more in other services. So I'm not sure if we can solve this cleanly with flags(types?) as we do in codecs.

Reply:
We discussed this offline and I made it into single functions, as you suggested. There is also second function that sets `available_contexts` - this must be done after connection exists and we know `conn_handle`.